css/index.css

```diff
@@ -172,4 +172,10 @@ header .logo {
 footer {
     text-align: center;
     margin: 40px 0 20px;
+}
+meta (max-width: 768px){
+    .container {
+        width: 100%; 
+        padding: 25px; 
```

Comment:
This is the right path to your project. I saw that in your latest commits that you were able to implement the solution code. I hope that it was helpful to you in being able to understand the material more in-depth. Keep up your awesome work and positive attitude! ❤️